test/services/FinancialInstitutionUpdateServiceSpec.scala

```diff
@@ -88,6 +95,73 @@ class FinancialInstitutionUpdateServiceSpec extends SpecBase with MockitoSugar w
       }
     }
 
+    "populateAndSaveRegisteredFiDetails" - {
+
+      "must populate and persist user answers with Reg FI details with different Reg Address" in {
+        val fiDetails       = testFiDetail
+        val testAddressResp = AddressResponse("22", Some("High Street"), Some("Dawley"), Some("Dawley"), Some("TF22 2RE"), "GB")
+        val country         = Country.GB
+        val ukCountryCodes  = Set(country.code, fiDetails.AddressDetails.CountryCode.value)
+        when(mockChangeUserAnswersRepository.get(any)).thenReturn(Future.successful(None))
```

Comment:
i dont think the the when().thenReturn() setup needs to be repeated for each test, you should be able to put then at the top of the hierarchy :)

Reply:
updated @C-Thomp  thanks